packages/common/src/lib/common/tools.ts

```diff
@@ -3212,7 +3212,7 @@ function removePreservedProperties(
  * @param namespace - adapter namespace + id, e.g. hm-rpc.0
  * @param createWakeup - indicator to create wakeup object too
  */
-function getInstanceIndicatorObjects(namespace: string, createWakeup: boolean): any[] {
+function getInstanceIndicatorObjects(namespace: string, createWakeup: boolean): Record<string, any>[] {
```

Comment:
We have proper types for every object type, I think this is the correct one:
```suggestion
function getInstanceIndicatorObjects(namespace: string, createWakeup: boolean): Record<string, ioBroker.StateObject>[] {
```

Reply:
yes, strange it already looks like this for a few commits